app/controllers/courses_controller.rb

```diff
@@ -656,6 +656,15 @@ def user_index
   #
   # @argument enroll_me [Boolean]
   #   Set to true to enroll the current user as the teacher.
+   #
```

Comment:
the extra space in front of this `#` seems to be confusing the parser :( _sigh_, would you mind fixing it?